### ISSUE TITLE
Nutritional plugin broken with SQLAlchemy 0.7 (& workaround)

### ISSUE THREAD

**Issue by ockham** [thinkle/gourmet#345]
Converted from [SourceForge issue 3568233](https://sourceforge.net/support/tracker.php?aid=3568233), submitted by [SourceForge user capned](https://sourceforge.net/users/?user_id=1712134) on 2012-09-16 21:05:43 UTC.

I did a new install of the latest Gourmet (0.15.9) on a linux machine that has never had either Gourmet or SQLAlchemy before. I installed the latest SQLAlchemy as well (0.7.8). Everything seemed to work fine, until I enabled the Nutritional Plugin. The download GUI appeared, and downloaded the USDA info database, then the next GUI loaded, and stalled. The progress bar never moved, and there were errors in the terminal where I started Gourmet. The error was from some path in the SQLAlchemy install path, saying, "immutabledict object is immutable". The last file in the stack trace with a gourmet path, I think, was data_plugin.py, under nutritional_information, line 68.

I think I was able to find a workaround: I installed the latest version of SQLAlchemy from the 0.6 series (0.6.9). Once I did that, and restarted gourmet, the plugin loaded. 

I previously had the same problem (though I didn't recognize it) on an Ubuntu machine, probably because SQLAlchemy updated and broke the nutritional plugin. Installing SQLAlcehmy 0.6.9 also fixed the problem on that box.

tl;dr: Nutritional plugin broken with SQLAlchemy 0.7, fixed by installing SQLAlchemy 0.6
